apis/Google.Cloud.Language.V1.Experimental/Google.Cloud.Language.V1.Experimental/project.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "1.0.0-beta00-*",
```

Comment:
I always update the version number *just* prior to releasing. Admittedly I'm not expecting the code to change before I do a release, but I thought I'd try to stick to the pattern.